src/components/NewLexemeForm.vue

```diff
@@ -46,20 +46,26 @@ export default {
 		<lexical-category-input
 			v-model="lexicalCategory"
 		/>
-		<wikit-button
-			class="form-button-submit"
-			type="progressive"
-			variant="primary"
-		>
-			{{ $messages.get( 'wikibaselexeme-newlexeme-submit' ) }}
-		</wikit-button>
+		<div>
+			<wikit-button
+				class="form-button-submit"
+				type="progressive"
+				variant="primary"
+			>
+				{{ $messages.get( 'wikibaselexeme-newlexeme-submit' ) }}
+			</wikit-button>
+		</div>
 	</form>
 </template>
 
 <style scoped lang="scss">
 @import "@wmde/wikit-tokens/variables";
 
 .wbl-snl-form {
+	// display
+	display: grid;
```

Comment:
Are you sure we can use grid? MDN says it’s supported since Safari 10.1, and [browser support matrix Grade A](https://www.mediawiki.org/wiki/Compatibility#Browser_support_matrix) (and [browserslist-config-wikimedia/modern](https://github.com/wikimedia/browserslist-config-wikimedia/blob/v0.3.0/modern.js#L9)) still include Safari 9.1 :/

Reply:
Our reference is [modern-es6-only](https://github.com/wikimedia/browserslist-config-wikimedia/blob/v0.3.0/modern-es6-only.js) as only that covers browsers that support the Proxy object and thus Vue 3. But that still includes Safari 10 and `display: grid` is apparently only supported in Safari 10.1 :persevere: 

I'll change it to flex-box :disappointed: 

/me should have been bold when creating that `modern-es6-only.js` file and trying to sneak in a _slightly_ newer Safari version :/

Reply:
updated. Please have another look

Reply:
Also ... I'm not sure vite actually makes use of `.browserlistrc`?

Reply:
I proposed a Codex change, it turns out to affect stylelint at least: https://gerrit.wikimedia.org/r/c/design/codex/+/763199